docs/docs-graphql/schema.graphqls

```diff
@@ -1534,6 +1534,7 @@ type Module {
   interfaces: [TypeDef!]!
   name: String!
   objects: [TypeDef!]!
+  runtime: Container!
```

Comment:
nit for a follow-up: why don't we get the doc for this?

Reply:
Huh, yeah weird, taking a look.